webroot/php/api/functions.inc

```diff
@@ -120,7 +120,7 @@ header("HTTP/1.0 500 Internal Server Error");
   exit(0);
 }
 
-function isValidCallback($callback) {
+function isValidCallback($callback) {

```

Comment:
This mush be a a trailing space, huh? Probably safe to assume the repo has a bunch of these.